### PR TITLE
ADX-249 add latest to image tag

### PR DESCRIPTION
**Pull request by fulior**
I'm trying to unify image tagging process, adding .latest and .timestamp suffixes

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -8,15 +8,17 @@ phases:
...
       - export BUILD_TAG=$(echo $CODEBUILD_WEBHOOK_TRIGGER | sed 's/.*\///g')
       - echo Build started on `date`
       - echo Building the Docker image...          
       - docker build -t datapusher:$BUILD_TAG .
-      - docker tag datapusher:$BUILD_TAG 254010146609.dkr.ecr.eu-west-1.amazonaws.com/datapusher:$BUILD_TAG
+      - docker tag datapusher:$BUILD_TAG 254010146609.dkr.ecr.eu-west-1.amazonaws.com/datapusher:$BUILD_TAG.latest
+      - docker tag datapusher:$BUILD_TAG 254010146609.dkr.ecr.eu-west-1.amazonaws.com/datapusher:$BUILD_TAG.${timestamp}
   post_build:
     commands:
+      - export timestamp=$(date +"%Y%m%dT%H%M%S")
```
Comment (tomeksabala) [inline buildspec.yml]: Do we even need the `timestamp` and `BUILD_TAG` here after changing the way we push datapusher images in line :24?

**Review by tomeksabala — CHANGES_REQUESTED** [review]

File: buildspec.yml (hunk 1 above)
Comment (fulior) [reply buildspec.yml]: you're right, we don't :)

**Comment by fulior** [comment]
Timestamp is now just a date, I've left BUILD_TAG but I'm using it to set the branch name only. I've disabled build on push as it was leaving artifacts in ECR that would be difficult to cleanup, no clean prefix there and it would trigger double build on each push to an open PR. So each build will now create an image with two tags: 

BRANCH_NAME:DATE & BRANCH_NAME:latest, e.g.:
development:20201115 & development:latest

**Review by tomeksabala — APPROVED** [review]
Looks good. I'll merge it right away as we'll be deploying new datapusher today.

**Pull request merged**